docs/Developer-Guide.md

```diff
@@ -219,15 +219,36 @@ rustup target add x86_64-unknown-linux-musl
 sudo ln -s /usr/bin/g++ /bin/musl-g++
 ```
 
+The agent is built with seccomp capability by default.
+Therefore, you **MUST** install `libseccomp` library and set environment variables for
+linking the agent binaries with `libseccomp` library statically before building the agent.
+
+To install `libseccomp` library:
+
+```
+$ $GOPATH/src/github.com/kata-containers/kata-containers/install_libseccomp.sh /tmp/kata-libseccomp /tmp/kata-gperf
+```
+
+> **Note**
+>
+> - You should build `libseccomp` library from sources using the above script to generate the static library for `musl`.
+
+To set environment variables for static link:
```

Comment:
Could you also document the dynamic link case and explain how to build an image with `libseccomp` in it?

Reply:
I added the descriptions